--- plugins/mcp-actions-backend/src/services/McpService.ts
+++ plugins/mcp-actions-backend/src/services/McpService.ts
@@ -34,7 +34,6 @@
   MetricsService,
   TracingService,
 } from '@backstage/backend-plugin-api/alpha';
-import { version } from '@backstage/plugin-mcp-actions-backend/package.json';
 import { NotFoundError, toError } from '@backstage/errors';
 import { performance } from 'node:perf_hooks';
 
@@ -81,6 +80,8 @@
   return attrs;
 }
 
+const version = '0.1.11'
+
 export class McpService {
   private readonly actions: ActionsService;
   private readonly logger: LoggerService | undefined;
