--- plugins/mcp-actions-backend/src/services/McpService.ts
+++ plugins/mcp-actions-backend/src/services/McpService.ts
@@ -32,7 +32,6 @@ import {
   MetricsService,
   TracingService,
 } from '@backstage/backend-plugin-api/alpha';
-import { version } from '@backstage/plugin-mcp-actions-backend/package.json';
 import { NotFoundError } from '@backstage/errors';
 import { performance } from 'node:perf_hooks';
 
@@ -79,6 +78,8 @@ function baggageAttributes(
   return attrs;
 }
 
+const version = '0.1.14'
+
 export class McpService {
   private readonly actions: ActionsService;
   private readonly logger: LoggerService | undefined;
