--- plugins/mcp-actions-backend/src/services/McpService.ts
+++ plugins/mcp-actions-backend/src/services/McpService.ts
@@ -34,7 +34,6 @@
   MetricsService,
   TracingService,
 } from '@backstage/backend-plugin-api/alpha';
-import { version } from '@backstage/plugin-mcp-actions-backend/package.json';
 import { NotFoundError, toError } from '@backstage/errors';
 import { performance } from 'node:perf_hooks';
 
@@ -42,6 +41,8 @@
 import { bucketBoundaries, McpServerOperationAttributes } from '../metrics';
 import { FilterRule, McpServerConfig } from '../config';
 
+const version = '0.1.11'
+
 function safeStringify(value: unknown): string {
   try {
     return JSON.stringify(value) ?? String(value);
